includes/class-wc-payments-captured-event-note.php

```diff
@@ -206,7 +206,7 @@ public function get_fee_breakdown() {
 
 		foreach ( $history as $fee ) {
 			$label_type = $fee['type'];
-			if ( isset( $fee['additional_type'] ) ) {
+			if ( isset( $fee['additional_type'] ) && ! empty( $fee['additional_type'] ) ) {
```

Comment:
Bonus: In the screenshot, you can also see that the base fee isn't introduced. There was a change in the server where `additional_type` is returned as `""` (empty string) instead of `null`, which executes the line after this.

Once executed, we'll be looking for a fee with `base-` as key => not available => no base fees reported

Reply:
Thanks for working on this. See my other comment/suggestion
https://github.com/Automattic/woocommerce-payments/pull/7024/files#r1300895708